kobo/apps/usage_statistics/admin.py

```diff
@@ -114,3 +186,4 @@ def __get_serialized_data(self, request) -> list:
 
 
 admin.site.register(KobocatSubmissionCounter, UserStatisticsAdmin)
+admin.site.register(Asset, SubmissionsByCountry)
```

Comment:
It would be better to register `ReadOnlyKobocatInstance` and change its verbose name and its app to `usage_statistic` to make it appear under the same section we created earlier for reports. 


FYI: PEP-8, you need a new line at the end the file. (This is can be set automatically with your IDE)